pilot/cmd/pilot-agent/status/server.go

```diff
@@ -83,6 +83,10 @@ var (
 
 	LegacyLocalhostProbeDestination = env.RegisterBoolVar("REWRITE_PROBE_LEGACY_LOCALHOST_DESTINATION", false,
 		"If enabled, readiness probes will be sent to 'localhost'. Otherwise, they will be sent to the Pod's IP, matching Kubernetes' behavior.")
+
+	ProbeKeepaliveConnections = env.RegisterBoolVar("PROBE_KEEPALIVE_CONNECTIONS", false,
```

Comment:
nit: maybe `ENABLE_...` so that it sounds more like a boolean?

Reply:
I agree and updated it